hs_restclient/__init__.py

```diff
@@ -1187,6 +1188,20 @@ def getResourceFolderContents(self, pid, pathname):
 
         return r.json()
 
+    def copyResource(self, pid):
```

Comment:
I was giving ```copyResource()``` as one example for making the API pattern look similar to the existing api. Either we implement these wrapper api functions for all the new api functions included in this PR or we should remove the copyResource().